getting-started/kubernetes/self-managed-onprem/onpremises.md

```diff
@@ -22,28 +22,28 @@ This how-to guide uses the following {{site.prodname}} features:
 
 #### {{site.prodname}} manifests
 
-{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. We recommend [Customizing Calico manifests ({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes; this avoids downstream manual updates to other {{site.prodname}} resources.
+{{site.prodname}} provides manifests for easy customization. Each manifest contains the necessary resources for installing {{site.prodname}} on each node in your Kubernetes cluster. You may want to [customize the {{site.prodname}} manifests]({{site.baseurl}}/getting-started/kubernetes/installation/config-options) before installing {{site.prodname}} on nodes.
 
 ### Before you begin...
 
 - Ensure that your Kubernetes cluster meets [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements).
   If you do not have a cluster, see [Using kubeadmin to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 - If you are using CoreOS, [make this required change to manifests]({{site.baseurl}}/reference/faq#are-the-calico-manifests-compatible-with-coreos).
 
-### How to 
+### How to
 
 - [Determine your datastore](#determine-your-datastore)
 - [Install Calico on nodes](#install-calico-on-nodes)
 
-#### Determine your datastore 
+#### Determine your datastore
 
-{{site.prodname}} supports both **Kubernetes API datastore (kdd)** and **etcd** datastores. The **Kubernetes API datastore** is preferred for on-premises deployments, and supports only Kubernetes workloads; **etcd** is the best datastore for hybrid deployments. 
+{{site.prodname}} supports both **Kubernetes API datastore (kdd)** and **etcd** datastores. The **Kubernetes API datastore** is the preferred for on-premises deployments, and supports only Kubernetes workloads; **etcd** is the best datastore for hybrid deployments.
```

Comment:
I think either:

The **Kubernetes API datastore** is preferred for on-premises deployments

or

The **Kubernetes API datastore** is the preferred datastore for on-premises deployments

are better.